src/GLESv1/Makefile.am

```diff
@@ -27,17 +27,24 @@
 
 lib_LTLIBRARIES = libGLESv1_CM.la
 
-ENTRYPOINT_TARGET=glesv1
-include $(top_srcdir)/src/OpenGL/entrypoint_common.mk
+EXTRA_libGLESv1_CM_la_DEPENDENCIES = $(builddir)/g_glesv1_exports.sym
 
-libGLESv1_CM_la_SOURCES = $(ENTRYPOINT_COMMON_FILES)
-
-libGLESv1_CM_la_CFLAGS = $(ENTRYPOINT_COMMON_CFLAGS)
+libGLESv1_CM_la_SOURCES = 
+libGLESv1_CM_la_LDFLAGS = -shared \
+	$(LINKER_FLAG_NO_UNDEFINED) \
+	-export-symbols $(builddir)/g_glesv1_exports.sym \
+	-version-info 1
 
-EXTRA_libGLESv1_CM_la_DEPENDENCIES = $(ENTRYPOINT_COMMON_DEPS)
```

Comment:
This makes $(ENTRYPOINT_COMMON_DEPS) unused, so it can be removed from src/OpenGL/entrypoint_common.mk.